elements/rh-secondary-nav/test/rh-secondary-nav.spec.ts

```diff
@@ -0,0 +1,185 @@
+import { expect, html, fixture, aTimeout, oneEvent } from '@open-wc/testing';
+import { setViewport } from '@web/test-runner-commands';
+
+import { RhSecondaryNav } from '../rh-secondary-nav.js';
+import { RhSecondaryNavDropdown } from '../rh-secondary-nav-dropdown';
+import { SecondaryNavDropdownChangeEvent } from '../rh-secondary-nav-dropdown.js';
+import { RhSecondaryNavOverlay } from '../rh-secondary-nav-overlay';
+
+import { NAV } from './fixtures';
+
+function isDesktop() {
+  const matches = !!window.matchMedia('(min-width: 992px)').matches;
+  return matches;
+}
+
+let element: RhSecondaryNav;
+let mobileButton: HTMLButtonElement | null | undefined;
+let overlay: RhSecondaryNavOverlay | null | undefined;
+let dropdown: RhSecondaryNavDropdown | null;
+
+describe('<rh-secondary-nav>', async function() {
+  beforeEach(async function() {
+    element = await fixture<RhSecondaryNav>(NAV);
+    mobileButton = element.shadowRoot?.querySelector('button');
+    overlay = element.shadowRoot?.querySelector('rh-secondary-nav-overlay');
+    dropdown = element.querySelector('rh-secondary-nav-dropdown a');
+  });
+
+  it('should upgrade', async function() {
+    const klass = customElements.get('rh-secondary-nav');
+    expect(element)
+      .to.be.an.instanceOf(klass)
+      .and
+      .to.be.an.instanceOf(RhSecondaryNav);
+  });
+
+  it('should remove role="navigation" after upgrade', async function() {
+    expect(element.hasAttribute('role')).to.be.false;
+  });
+
+  it('should have an overlay set to hidden after upgrade', async () => {
```

Comment:
```suggestion
  it('should have an overlay set to hidden after upgrade', async function() {
```

prefer [function keyword in mocha tests](https://mochajs.org/#arrow-functions)

Reply:
Fixed in: de6c9fedc09782910db0fe477bb6879c55876c26